src/Float.mo

```diff
@@ -471,47 +509,63 @@ module {
   /// ```
   public let fromInt : Int -> Float = Prim.intToFloat;
 
-  /// Returns `x == y`.
+  /// Determines whether `x` is equal to `y` within the defined tolerance of `epsilon`.
+  /// The `epsilon` considers numerical erros, see comment above.
+  /// Equivalent to `Float.abs(x - y) <= epsilon` for a non-negative epsilon.
   ///
-  /// Note: This operation is discouraged as it does not consider numerical errors, see comment above.
+  /// Traps if `epsilon` is negative or `nan`.
```

Comment:
```suggestion
  /// Traps if `epsilon` is negative or `NaN`.
```

Reply:
`NaN` is the canonical spelling, there are a few more `nan` in comments, can you change those too?